test/ctia/domain/id_test.clj

```diff
@@ -0,0 +1,73 @@
+(ns ctia.domain.id-test
```

Comment:
Notice the mixture of specs and tests in this file.  It works well because there are no fixtures and all of the functions being tested are referentially transparent.